client/src/helpers/storeSearch.ts

```diff
@@ -0,0 +1,340 @@
+import { DEFAULT_NGRAM_LENGTH } from "src/constants";
+import { Store } from "src/interfaces";
+import { Heap } from "src/packages/Heap";
+
+/**
+ * Preprocesses a string s to use as input for calculateEditDistance().
```

Comment:
Add a bit more explanation how the string is being pre-processed e.g. Removing quotation such as ",", removing conjunctions such as "&", "and" etc.